WaterNetworkManagement/Simulation/SimulationScenario/examples/example-normalized-ld.jsonld

```diff
@@ -0,0 +1,190 @@
+{
+    "id": "urn:ngsi-ld:SimulationScenario:01",
+    "type": "SimulationScenario",
+    "description": {
+        "type": "Property",
+        "value": "Free Text"
+    },
+    "createdBy": {
+        "type": "Relationship",
+        "object": "urn:ngsi-ld:User:u1"
+    },
+    "hasInputNetwork": {
+        "type": "Relationship",
+        "object": "urn:ngsi-ld:WaterNetwork:01"
+    },
+    "hasSimulationResult": {
+        "type": "Relationship",
+        "object": "urn:ngsi-ld:Result:01"
+    },
+    "duration": {
+        "type": "Property",
+        "value": 86400,
+        "unitCode": "SEC"
+    },
+    "hydraulicTimeStep": {
+        "type": "Property",
+        "value": 3600,
+        "unitCode": "SEC"
+    },
+    "flowUnits": {
+        "type": "Property",
+        "value": "LPS"
+    },
+    "headlossFormula": {
+        "type": "Property",
+        "value": "H-W"
+    },
+    "demandModel": {
+        "type": "Property",
+        "value": "PDA"
+    },
+    "operationalControl": [
+        {
+            "type": "Property",
+            "value": "Operational Control 1",
+            "setting": {
+                "type": "Property",
+                "value": 0
+            },
+            "triggerLevel": {
+                "type": "Property",
+                "value": 30
+            },
+            "controlType": {
+                "type": "Property",
+                "value": "HILEVEL"
+            },
+            "controlledLink": {
+                "type": "Relationship",
+                "object": "urn:ngsi-ld:Tank:T1",
+                "datasetId": "urn:ngsi-ld:Dataset:Control01:Node01"
+            },
+            "monitoredNode": {
+                "type": "Relationship",
+                "object": "urn:ngsi-ld:Pump:P1",
+                "datasetId": "urn:ngsi-ld:Dataset:Control01:Link01"
+            },
+            "datasetId": "urn:ngsi-ld:Dataset:HiLevel"
+        },
+        {
+            "type": "Property",
+            "value": "Operational Control 2",
+            "triggerLevel": {
+                "type": "Property",
+                "value": 10
+            },
+            "setting": {
+                "type": "Property",
+                "value": 1
+            },
+            "controlType": {
+                "type": "Property",
+                "value": "LOWLEVEL"
+            },
+            "monitoredNode": {
+                "type": "Relationship",
+                "object": "urn:ngsi-ld:Tank:T1"
+            },
+            "controlledLink": {
+                "type": "Relationship",
+                "object": "urn:ngsi-ld:Pump:P1"
+            },
+            "datasetId": "urn:ngsi-ld:Dataset:LowLevel"
+        }
+    ],
+    "inputParameters": [
```

Comment:
Format of the the first example here is different to the others. Where have `applyToEntity` and `forAttribute` come from?

Reply:
yes this was another proposition. 
updated